v63004/u_inref/gtm8922.csh

```diff
@@ -25,6 +25,8 @@ $MUPIP set -access_method=BG -file mumps.dat >>& dbcreate_log.txt
 $MUPIP set -access_method=BG -file a.dat >>& dbcreate_log.txt
 $MUPIP set -access_method=BG -file b.dat >>& dbcreate_log.txt
 
+$MUPIP set -file mumps.dat -flush_time=1:0:0 # Prevent interruptions from flush timers
```

Comment:
The flush timer needs to be set to a high value on ALL regions ("*"), not just DEFAULT. This is a test timing issue waiting to happen in other regions too.